utils/user-product-helper.ts

```diff
@@ -1,15 +1,17 @@
 import { ProductCode } from '@/enums'
 import { ProductI } from '@/interfaces'
+import { appendAccountId } from 'sbc-common-components/src/util/common-util'
 
 /**
  * Returns product info object for specified type.
  */
 export function getProductInfo (config, type: ProductCode): ProductI {
   switch (type) {
     case ProductCode.BUSINESS:
+      const businessLink = appendAccountId(config?.myBusinessRegistryDashboard) 
```

Comment:
This seems to work for SBC Staff logins but I think it's not needed (since they don't have multiple accounts). Anyway, I made it a separate variable here:
- for cleaner code
- in case we need to add logic in the future